app/src/main/java/org/schabi/newpipe/settings/SettingsResourceRegistry.java

```diff
@@ -29,6 +29,10 @@ public final class SettingsResourceRegistry {
     private final Set<SettingRegistryEntry> registeredEntries = new HashSet<>();
 
     private SettingsResourceRegistry() {
+        add(MainSettingsV2Fragment.class, R.xml.main_settings_v2).setSearchable(false);
+
+        // Before redesign settings arrangement. These should be cleared once the
```

Comment:
Thanks for your interest in testing this feature :tada:  All work related to the redesigned settings page is tracked in Issue #9587.

#### Testing Instructions:
- The testing instructions for the settings page update are the same as those provided in the pull request description.

#### Current Implementation to be Superseded
- Based on an offline conversation and the planned rewrite discussed in #10118, the new changes will utilize the new UI framework, Jetpack Compose, to achieve the same functionality.
- So the current PR will be discarded at some point.

#### Hold Off on Testing for Now
- As this feature is in a very early stage, and the current changes are only related to the backend and framework work (rather than user-facing changes), I recommend holding off on testing until the initial PRs get merged.
- The testing instructions will remain the same, regardless of the implementation chosen.